src/content/guides/how-to-revoke-token-access/index.md

```diff
@@ -0,0 +1,64 @@
+
+Sometimes malicious developers build backdoors into smart contracts that will allow access to the funds of unaware users who interact with their projects. What often happens is that such platforms ask the user for permission to spend an **unlimited number of tokens**.
+
+Once a platform has unlimited access, they can spend all your tokens even if you have withdrawn your funds from their platform into your wallet. Malicious actors can still access your funds and withdraw them into their wallet with no recovery option left for you. 
+
+The only protection is not to use untested new projects or regularly revoke access. So how to do it?
+
+## Step 1: Use revoke access tools
+
+There are several websites that let you view and revoke smart contracts connected to your address. Visit the website and connect your wallet:
+
+- [Ethallowance](https://ethallowance.com/) (Ethereum)
+- [Etherscan](https://etherscan.io/tokenapprovalchecker) (Ethereum)
+- [Cointool](https://cointool.app/approve/eth) (multiple networks)
+- [Revoke](https://revoke.cash/) (multiple networks)
+- [Unrekt](https://app.unrekt.net/) (multiple networks)
+- [EverRevoke](https://everrise.com/everrevoke/) (multiple networks)
+
+## Step 2: Connect your wallet
+
+Once you are on the website, click on “Connect wallet”. The website should prompt you to connect your wallet.  
+
+Make sure you are using the right network in your wallet and on the website. You will only see smart contracts related to the network selected. For example: if you are connected to Ethereum, you will only see Ethereum contracts and not contracts from other chains like Layer 2s.
+
+## Step 3: Select a smart contract you wish to revoke
+
+Now you should see all the contracts that are allowed access to your tokens and their potential spending limit.  Locate the one you wish to terminate.
+
+If you do not know which contract to choose, you can revoke all of them. This should not create any major issues for you. It will just mean that the next time you visit any project and wish to interact with it, you will have to grant a new set of permissions. 
+
+## Step 4: Revoke access to your funds
+
+Once you click on revoke, you should see a new transaction suggestion in your wallet. This is to be expected. You will have to pay the fee for the cancelation to be successful. Depending on the network this can take from a minute to several to be processed.
+
+We advise to refresh the revoking tool after few minutes and connect your wallet again to double check if revoked contract has disappeared from the list. 
+
+<mark> We reccomend that you never allow projects unlimited access to your tokens and that you revoke all token allowance access regularly every month. </mark>
+
+### FAQ:
+
+**Does revoking token access also terminate staking, pooling, lending etc?**
+
+- No it will not affect any of your DEFI strategies. You will remain your positions and keep getting rewards etc.
```

Comment:
These shouldn't use lists if they are single statements.

Reply:
@minimalsm do we have any plug and  play component for FAQs here or should this be just plain text?

Reply:
or @wackerow what would be your suggestion? just text without any bullet point?